doc/lua_api.md

```diff
@@ -9381,6 +9381,13 @@ Used by `minetest.register_node`.
     -- (see `liquid_move_physics`), the movement speed will also be
     -- affected by the `movement_liquid_*` settings.
 
+    climb_factor = 1.0,
+    -- The speed at which a climbable node can be climbed is multiplied
+    -- with this number. Must not be negative. No effect if node isn't
+    -- climbable.
```

Comment:
up,down and s/with/by/
```suggestion
    -- The speed at which a climbable node can be climbed up and down
    -- is multiplied by this number. Must not be negative. No effect if
    -- node isn't climbable.
```

Might also be worth noting here that climb speed depends on horizontal walking speed.